cmd/appgw-ingress/main.go

```diff
@@ -11,13 +11,18 @@ import (
 	"os"
 	"time"
 
+	"k8s.io/client-go/kubernetes/scheme"
+	"k8s.io/client-go/tools/record"
+
```

Comment:
Should we delete the empty line here and then sort the second half of the imports?